nixos/tests/limine/checksum.nix

```diff
@@ -0,0 +1,33 @@
+import ../make-test-python.nix (
+  { lib, ... }:
+  {
+    name = "limine-checksum";
+    meta.maintainers = with lib.maintainers; [
+      lzcunt
+      phip1611
+      programmerlexi
+    ];
+    meta.platforms = [
+      "aarch64-linux"
+      "i686-linux"
+      "x86_64-linux"
+    ];
+    nodes.machine =
+      { ... }:
+      {
+        virtualisation.useBootLoader = true;
+        virtualisation.useEFIBoot = true;
+
+        boot.loader.efi.canTouchEfiVariables = true;
```

Comment:
Set installEfi to true here, and maybe a BIOS test would be beneficial. Not sure if this would work without canTouchEfiVariables - it should on sane firmware AFAIK. It's probably best to keep it. Applies to the other test as well

Reply:
I've been trying to add a BIOS test for the past 2 hours.
I can't write a test for that.

Reply:
As far as I can tell, the main issue is that by default, `vmWithBootLoader` only adds a separate boot partition when set to EFI mode, where as Limine requires a separate boot partition for both BIOS and EFI booting. A BIOS test for Limine would require getting NixOS to generate a boot partition in spite of EFI being disabled.